server/datastore/mysql/packs.go

```diff
@@ -254,41 +280,41 @@ func replacePackTargetsDB(ctx context.Context, tx sqlx.ExecerContext, pack *flee
 		return ctxerr.Wrap(ctx, err, "delete pack targets")
 	}
 
-	// Insert hosts
-	if len(pack.HostIDs) > 0 {
+	// Insert labels
+	if len(pack.LabelIDs) > 0 {
```

Comment:
I'm curious, it seems here the insertion order of hosts and labels have just been swapped? Was there any reason for that?